modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -91,6 +93,19 @@ def veteran_from_headers(with_gender: false)
         vet
       end
 
+      def validate_ccg_token!
+        client = TokenValidation::V2::Client.new(api_key: Settings.token_validation.api_key)
+        request_method_to_scope = {
+          'GET' => 'claim.read',
+          'PUT' => 'claim.write',
+          'POST' => 'claim.write'
+        }
+        raise ::Common::Exceptions::Forbidden unless
+          client.token_valid?(audience: "#{request.base_url}/services/claims",
```

Comment:
`request.base_url` probably needs to adjust for local requests, right? localhost won't work for this since it's hosted separately

Reply:
Yeah.  If you're running this locally, you'd need to replace `request.base_url` with the url of whatever environment you're trying to talk to.